Resources/public/js/views/services/plugins/ez-savedraftplugin.js

```diff
@@ -146,9 +173,11 @@ YUI.add('ez-savedraftplugin', function (Y) {
          * @method _createContent
          * @param Array fields the fields structures coming from the saveAction
          * event
+         * @param Function serverSideErrorCallback callback defined in ez-contenteditview which will
```

Comment:
you should not document where the callback is defined nor what it does (at some point this will be wrong) but what is its purpose so here:
```
@param {Function} serverSideErrorCallback called when a server side validation is detected
```